hivemind/optim/experimental/grad_averager.py

```diff
@@ -0,0 +1,219 @@
+import contextlib
+from typing import Iterable, Iterator, Optional
+
+import torch
+
+import hivemind.dht.dht
```

Comment:
Right now this import seems unused: it's better to directly `from hivemind.dht import DHT` if you need it to avoid import scope creep and side effects

Reply:
fixed